.github/workflows/preview.yml

```diff
@@ -0,0 +1,228 @@
+# +-----------------------------------------------+
+# | WARNING: This file is generated, do not edit!  |
+# | Edit _templates/distribution-channel.yml.erb instead.        |
+# +-----------------------------------------------+
+
+name: Build and push images (preview)
+on:
+  push:
+    branches:
+    - master
+    paths:
+    - "**/*"
+    - "!README.md"
+    - "!build.rb"
+  schedule:
+    # As well as running when we make changes we should run at least
+    # every week in order to pick up new parent images and new versions of Snyk
+    - cron:  "0 0 * * 0"
+  workflow_dispatch:
+  repository_dispatch:
+    types: [build_and_push_images]
```

Comment:
Issue: The repository dispatch type also needs the postfix, so that webhook triggers can build the different channels and not always build all.

Reply:
Good catch, updated here to introduce a filter that will match on the supplied `release_channel` value from the Snyk CLI repo. 
https://github.com/snyk/cli/pull/5654/files